crates/core/tedge_mapper/src/mapping/converter.rs

```diff
@@ -18,10 +20,10 @@ pub trait Converter: Send + Sync {
         &self.get_mapper_config().in_topic_filter
     }
 
-    fn try_convert(&mut self, input: &Message) -> Result<Vec<Message>, Self::Error>;
+    async fn try_convert(&mut self, input: &Message) -> Result<Vec<Message>, Self::Error>;
```

Comment:
As discussed, it's okay for now to make `async` this method. But this will have to be reverted when the operation execution will be delegated to the agent. Then a converter should only translate messages with any sidde effects.